<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="features">
   <set>QDockWidget::NoDockWidgetFeatures</set>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <item row="1" column="0" colspan="3">
     <widget class="QTreeWidget" name="workspacesList">
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="headerStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">Workspace/curve</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">Remove</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="addButton">
      <property name="minimumSize">
       <size>
        <width>27</width>
        <height>27</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Add the selected workspace to the list</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="WorkspaceSelector" name="workspaceSelector">
      <property name="toolTip">
       <string>Select a workspace</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WorkspaceSelector</class>
   <extends>QComboBox</extends>
   <header>mantidqt.widgets.workspaceselector</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
